samples/LocalEnvSecretStoreSample/Services/LocalEnvSecretStore.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Dapr.PluggableComponents.Components;
+using Dapr.PluggableComponents.Components.SecretStores;
+
+namespace LocalEnvSecretStoreSample.Services;
+
+internal sealed class LocalEnvSecretStore : ISecretStore
+{
+    private readonly ILogger<LocalEnvSecretStore> logger;
+
+    public LocalEnvSecretStore(ILogger<LocalEnvSecretStore> logger)
+    {
+        this.logger = logger;
+    }
+
+    #region ISecretStore Members
+
+    public Task<SecretStoreGetResponse> GetAsync(SecretStoreGetRequest request, CancellationToken cancellationToken = default)
+    {
+        this.logger.LogInformation("Get request for secret {key}", request.SecretName);
+
+        SecretStoreGetResponse? response = null;
+        string data = Environment.GetEnvironmentVariable(request.SecretName);
```

Comment:
Nit: should be `string? data = ...`?

I'd expect a build warning about nullability otherwise (as `GetEnvironmentVariable()` returns `string?`).

Reply:
`GetEnvironmentVariable ` returns string when environment variable is not found or set.

Reply:
`GetEnvironmentVariable()` returns `string?` (a potentially `null` string).  An alternative would be:

```csharp
string data = Environment.GetEnvironmentVariable() ?? String.Empty;
```

...which assigns `""` only if the return value is `null`.